.github/workflows/scancode-release.yml

```diff
@@ -275,7 +282,7 @@ jobs:
       fail-fast: true
       matrix:
        os: [windows-2019, windows-2022]
-       pyver: ["3.7", "3.8", "3.9", "3.10"]
+       pyver: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
We do smoke tests for installing and running scans with pypi wheels in windows and linux but not macos? Any reason we shouldn't do the same for mac?

Reply:
Actually there's macos under posix, resolving this.